src/modules/components/proposal/proposalActions/actions/proposalActionTokenMint/index.ts

```diff
@@ -0,0 +1,2 @@
+export { ProposalActionTokenMint } from './proposalActionTokenMint';
+export type { IProposalActionTokenMintProps } from './proposalActionTokenMint';
```

Comment:
You can have both on the same export:
```suggestion
export { ProposalActionTokenMint, type IProposalActionTokenMintProps } from './proposalActionTokenMint';
```